src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -65,6 +65,12 @@
              "")
            ""))))
 
+(defn get-crypto-decimals-count
+  [{:keys [market-values-per-currency]}]
+  (let [price          (get-in market-values-per-currency [:usd :price])
+        one-cent-value (if (pos? price) (/ 0.01 price) 0)]
```

Comment:
Yeah, I think `price` should be. 

JS numbers' implementation sometimes have an unexpected behaviour. since we are dealing with money, better to be safe :+1:

Reply:
Yeah, I think price should be.

JS numbers' implementation sometimes have an unexpected behaviour. since we are dealing with money, better to be safe 👍